libsolidity/formal/CHC.cpp

```diff
@@ -254,16 +254,17 @@ void CHC::endVisit(FunctionDefinition const& _function)
 	// Query placeholders for constructors are not created here because
 	// of contracts without constructors.
 	// Instead, those are created in endVisit(ContractDefinition).
-	if (!_function.isConstructor())
+	if (
+		!_function.isConstructor() &&
```

Comment:
```suggestion
		m_currentContract->canBeDeployed() &&
		!_function.isConstructor() &&
```

Reply:
Maybe we should do more for abstract contracts. So this may not be right change, in the end.